components/AdvancedOptionsItem.qml

```diff
@@ -0,0 +1,184 @@
+import QtQuick 2.9
+import QtQuick.Layouts 1.1
+import FontAwesome 1.0
+
+import "../components" as MoneroComponents
+
+RowLayout {
+    id: advancedOptionsItem
+    
+    property alias title: title.text
+    property alias button1: button1
+    property alias button2: button2
+    property alias button3: button3
+    property alias button4: button4
```

Comment:
Also unused. If people need 4 buttons at some point they can update the component.